api/src/main/scala/za/co/absa/loginsvc/rest/SecurityConfig.scala

```diff
@@ -58,13 +61,20 @@ class SecurityConfig @Autowired()(authConfigsProvider: AuthConfigProvider) {
     if(ldapConfig != null)
       {
         if(ldapConfig.enableKerberos.isDefined)
-          {
-            val kerberos = new KerberosSPNEGOAuthenticationProvider(ldapConfig)
+        {
+          val kerberos = new KerberosSPNEGOAuthenticationProvider(ldapConfig)
 
-            http.addFilterBefore(
+          http.addFilterBefore(
               kerberos.spnegoAuthenticationProcessingFilter,
               classOf[BasicAuthenticationFilter])
-          }
+              .exceptionHandling()
+              .authenticationEntryPoint((request: HttpServletRequest,
+                                         response: HttpServletResponse,
+                                         authException: AuthenticationException) => {
+                response.setStatus(HttpServletResponse.SC_UNAUTHORIZED)
+                response.addHeader("WWW-Authenticate", "Negotiate")
```

Comment:
I tried it and when kerberos was disabled in config, running a POST against the endpoint ( `curl -X POST http://localhost:9090/token/generate -v`) resulted only in:
`< WWW-Authenticate: Basic realm="Realm"`
(as expected ✅ ).

However, when running the same while having kerberos enabled in config, it only returned
`< WWW-Authenticate: Negotiate`
This time I would expect both options being present in the headers:
`< WWW-Authenticate: Basic realm="Realm"`
`< WWW-Authenticate: Negotiate`

Reply:
Hi Daniel

I see the issue. I was testing and accidentally left -u in the curl command and that would get WWW-Authenticate: Basic realm="Realm" back along with the Negotiate. So it looked like it was working properly on my end. Will sort this out along with the OpenAPI.